src/Components/Topics/Vote.jsx

```diff
@@ -0,0 +1,54 @@
+import { useState } from 'react'
+import { patchVoteById } from '../API/Api'
+import Button from './Button'
+import styles from './Vote.module.css'
+
+function Vote(props) {
+    const [disableLikeButton, setDisableLikeButton] = useState(false)
+    const [disableUnLikeButton, setDisableUnLikeButton] = useState(true)
+    const [error, setError] = useState(false)
+
+    function VoteHandlerApi(voteCount) {
+        setDisableLikeButton((prevState) => !prevState)
+        if (voteCount === 1) {
+            props.setVotes((prevVote) => prevVote + 1)
+        } else {
+            props.setVotes((prevVote) => prevVote - 1)
+        }
+        patchVoteById(props.article.article_id, voteCount).catch((err) => {
+            setError(true)
+        })
+        setDisableUnLikeButton((prevState) => !prevState)
+    }
+
+    return (
+        <div className={styles.vote}>
+            <p>
+                <b>CLICK TO VOTE</b>
+            </p>
+            <Button
+                disabled={disableLikeButton ? 'disabled' : ''}
+                onClick={() => VoteHandlerApi(+1)}
+            >
+                <img
+                    className={styles.like}
+                    alt="like"
+                    src={require('../Tools/LikeSVG/likeicon.png')}
+                />
```

Comment:
you're mixing import styles in this file which is a bit unusual

You should only be using `import * from './path'` module syntax 

you can import this image at the top with the rest of your imports and then put it in `src={}`